### PR TITLE
Patch damage and healing types config, add color, label, isPhysical properties

### PR DESCRIPTION
**Pull request by krbz999**

### CODE REVIEW

**Review by arbron — CHANGES_REQUESTED** [review]
Looks good, but pull out the colors for now to handle in a separate PR with a proposed usage. Let's just do the change to an object in this one.

**Comment by Fyorl** [comment]
> Bruh.

Well, for the separate PR we can consider using the `Color` type (:

**Review by arbron — APPROVED** [review]

**Comment by arbron** [comment]
<too-many-cooks.gif>

File: module/config.mjs [hunk 1]
```diff
@@ -1186,35 +1187,54 @@ preLocalize("physicalDamageTypes", { sort: true });
 
 /**
  * Types of damage the can be caused by abilities.
- * @enum {string}
+ * @enum {{label: string, color: string}}
```
Comment (Fyorl) [inline module/config.mjs:1190]:
```suggestion
 * @enum {{label: string, [isPhysical]: boolean}}
```

**Review by Fyorl — APPROVED** [review]

**Pull request merged**